ov/scope.go

```diff
@@ -0,0 +1,195 @@
+package ov
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/rest"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"github.com/docker/machine/libmachine/log"
+)
+
+type Scope struct {
+	Description utils.Nstring `json:"description,omitempty"` // "description": "Scope 1",
+	//Modified              string        `json:"modified,omitempty"`              // "modified": "20150831T154835.250Z",
+	Name string `json:"name,omitempty"` // "name": "Ethernet Network 1",
+	//State                 string        `json:"state,omitempty"`                 // "state": "Normal",
```

Comment:
remove comment out fields if they are not required

Reply:
updated the structure members